wiki/Guides/BSS_issues/en.md

```diff
@@ -4,35 +4,39 @@ tags:
   - beatmap submission system issues
 ---
 
+::: Infobox
+![Screenshot of the warning message](img/errormessage.png "The warning message as shown in the editor")
```

Comment:
The info box looks *really* wacky when being at the very top of the page. See the [preview](https://osu1.roanh.dev/wiki/en/Guides/BSS_issues). Would suggest having it right after `How to resolve BSS issues`, but not before.